### PR TITLE
[Icon] Add Inactive location icons

### LINKED ISSUE #7502: [Icon]: New icons <Inactive location>
## New Icon summary

We're merging the deactivated and active locations lists on the Locations settings page, so we're aiming to add a new icon to Polaris that can be used for locations that have been deactivated. This icon will allow merchants to easily scan and know what locations aren't in use across their store. 

_This icon is live in the admin, but we'd like to officially add it to polaris._

Screen Shot 2022-10-25 at 11 17 29 AM

## Icon name
Inactive location

## What type of icon set?
Both Major and Minor

## Icon SVG code

**Inactive location Minor SVG:** 
locations-inactive_minor

**Inactive location Major SVG:** 
location-inactive_major

### PR DESCRIPTION
**Pull request by rdott**
### WHY are these changes introduced?

Resolves: https://github.com/Shopify/polaris/issues/7502

Adds `Major` and `Minor` icons for Inactive Location.

Web admin is currently using this icon in the Location Settings List, and now adding this so it can be used directly from Polaris.

[Slack discussion](https://shopify.slack.com/archives/C4Y8N30KD/p1667240213636509)

### WHAT is this pull request doing?

Added major and minor version for new `Inactive Location` icon.

### CODE REVIEW

**Review by heyjoethomas — APPROVED** [review]
Looks good to me. Thanks for the contribution. If you need any help adding these to Figma let me know.

File: .changeset/violet-taxis-pump.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-icons': minor
+---
+
+Add inactive location minor and major icons
```
Comment (kyledurand) [inline .changeset/violet-taxis-pump.md]:
```suggestion
Added inactive location minor and major icons
```

**Review by kyledurand — APPROVED** [review]
Code looks good 👍

**Comment by ShaniqueS** [comment]
> Looks good to me. Thanks for the contribution. If you need any help adding these to Figma let me know.

@heyjoethomas I've [added a branch ](https://www.figma.com/file/mMHFt3kEDNjLMZWowi6gnt/branch/gzwUmR4Cu31cK3L67tQqIb/Polaris-Icons) to the Figma and have requested a review from Sara, but if you're also able to review then I can add you as well. 😄

**Pull request merged**